config/betamocks/services_config.yml

```diff
@@ -934,3 +934,32 @@
     - :method: :post
       :path: "/sts/oauth/v1/token"
       :file_path: "map/secure_token_service/token"
+
+  # Sign Up Service API
+- :name: "MAP SUS"
+  :base_uri: <%= "#{URI(Settings.mobile_application_platform.sign_up_service_url).host}:#{URI(Settings.mobile_application_platform.sign_up_service_url).port}" %>
+  :endpoints:
+    - :method: :get
+      :path: "/signup/v1/patients/*/status/summary"
+      :file_path: "map/sign_up_service/status"
+      :cache_multiple_responses:
+        :uid_location: url
+        :uid_locator: '\/signup\/v1\/patients\/(\d{10}[V]\d{6})\/status\/summary'
```

Comment:
I would make the capture regex for the icn just anything in between `patients` and `status`.  Yours might be more accurate but it will unexpectedly attempt to mock with a file called `.yml` if the given ICN doesn't exactly match 10 digits, 'V', 6 digits

Reply:
(And do something similar for the rest of these)